lib/cli/placeorder.ts

```diff
@@ -2,11 +2,13 @@ import { Arguments, Argv } from 'yargs';
 import { Order, OrderSide, PlaceOrderEvent, PlaceOrderRequest, PlaceOrderResponse, SwapFailure, SwapSuccess } from '../proto/xudrpc_pb';
 import { callback, loadXudClient } from './command';
 import { coinsToSats, satsToCoinsStr } from './utils';
+import { checkDecimalPlaces } from '../utils/utils';
 
 export const placeOrderBuilder = (argv: Argv, side: OrderSide) => {
   const command = side === OrderSide.BUY ? 'buy' : 'sell';
   argv.option('quantity', {
     type: 'number',
+    describe: 'the quantity for limit/market orders',
```

Comment:
We have some order besides limit and market, so I'd either leave this line out (not needed for this PR anyhow) or just say something more generic like "the quantity to trade".

Reply:
Done